sdk/tracing/azure-core-tracing-opencensus/src/test/java/com/azure/core/tracing/opencensus/OpenCensusTracerTest.java

```diff
@@ -51,27 +183,92 @@ public void startSpanOverloadNullPointerException() {
         openCensusTracer.start("", Context.NONE, null);
     }
 
+    //add tests for number of child spans created and how parent span relation
+
+    @Test
+    public void addLinkTest() {
+        // Arrange
+        // Create a child-parent link between multiple spans
+        RecordEventsSpanImpl existingSpan = (RecordEventsSpanImpl) tracer.spanBuilder("existing-span").startSpan();
+        Context traceContext = tracingContext.addData(SPAN_CONTEXT, existingSpan.getContext());
+        RecordEventsSpanImpl parentSpanImpl = (RecordEventsSpanImpl) parentSpan;
+
+        // Act
+        openCensusTracer.addLink(traceContext);
+
+        //Assert
+        // TODO: existing -> parentSpanImpl comes after after add Link, confirm this behavior?
+        Assert.assertEquals(parentSpanImpl.getPrev(), existingSpan);
+        Assert.assertEquals(parentSpanImpl.toSpanData().getChildSpanCount().intValue(), 1);
+        Assert.assertEquals(parentSpanImpl.toSpanData().getLinks().getLinks().size(), 1);
+    }
+
+    @Test
+    public void endSpanNoSuccessErrorMessageTest() {
+        // Arrange
+        RecordEventsSpanImpl recordEventsSpan = (RecordEventsSpanImpl) tracer.getCurrentSpan();
+
+        // Act
+        openCensusTracer.end(null, null, tracingContext);
+
+        // Assert
+        Assert.assertEquals(recordEventsSpan.getStatus().getCanonicalCode().toString(), "UNKNOWN");
```

Comment:
I know this is test code, but consider moving the string literals into a constant